include/grpc/grpc.h

```diff
@@ -351,6 +351,23 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
+/** Propagate census context */
+#define GRPC_PROPAGATE_CENSUS_CONTEXT ((gpr_uint32)2)
```

Comment:
> I don't suppose you'd want to send me a PR against this one implementing it? :)

I can - but I think it will have to wait till we get #2668 figured out to do it "right". Which reminds me, I need to go and update that PR...